app/src/main/java/uw/hcrlab/kubi/lesson/FlashCard.java

```diff
@@ -0,0 +1,130 @@
+package uw.hcrlab.kubi.lesson;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.nostra13.universalimageloader.core.ImageLoader;
+
+import uw.hcrlab.kubi.R;
+
+public class FlashCard extends LinearLayout implements View.OnTouchListener {
+    private static String TAG = FlashCard.class.getSimpleName();
+
+    private PromptData.Option option;
+
+    private boolean selected = false;
+
+    private OnFlashCardSelectedListener parent;
+
+    public interface OnFlashCardSelectedListener {
+        void onFlashCardSelected(FlashCard view);
+    }
+
+    public FlashCard(Context context) {
+        super(context);
+        initializeViews(context);
+    }
+
+    public FlashCard(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        initializeViews(context);
+    }
+
+    public FlashCard(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        initializeViews(context);
+    }
+
+    private void initializeViews(Context context) {
+        this.setId(View.generateViewId());
+
+        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        inflater.inflate(R.layout.flash_card_layout, this);
+
+        this.setOnTouchListener(this);
+    }
+
+    /* Should be called before onCreateView(). Not using setArguments because
+     * it only allows strings. */
```

Comment:
Is this comment still applicable now that we're using a ListAdapter rather than a Fragment?